test/e2e/play_kube_test.go

```diff
@@ -5245,6 +5250,44 @@ ENV OPENJ9_JAVA_OPTIONS=%q
 		deleteAndTestSecret(podmanTest, "newsecret")
 	})
 
+	It("secret as volume support - error on invalid secret format", func() {
+
```

Comment:
Nit: this starting whitespace should be removed - but only do this if you need to re-push for another reason, not worth it otherwise